components/inputs/input-time.js

```diff
@@ -118,15 +131,19 @@ class InputTime extends LitElement {
 		this.labelHidden = false;
 		this.timeInterval = 'thirty';
 		this._dropdownId = getUniqueId();
-		this._formattedValue = formatTime(DEFAULT_VALUE);
 		this._timezone = formatTime(new Date(), {format: 'ZZZ'});
-		this._value = formatValue(DEFAULT_VALUE);
 	}
 
 	get value() { return this._value; }
 	set value(val) {
+		if (this.value === undefined) {
+			this._givenValue = val;
+			return;
```

Comment:
Trying to figure out what this is doing... so `_givenValue` is tracking the first ever provided `value`? And then later in `firstUpdated` if that first-ever-value is undefined (or empty), it'll set `value` to default otherwise use the value they provided. Why can't we just do that here in the setter? Sorry if we covered that in the first round.

Reply:
To sum up a slack convo for posterity: this made sense when `parseValue` gave `0:00:00` as a default instead of throwing (had to check for value and default value at the same time to prevent overwriting things) but now that value can be checked on its own first it can be done here. Fixed :)